src/hatti/map/viewby.cljs

```diff
@@ -37,7 +37,9 @@
     (form-utils/text? field) raw-answers
     (form-utils/calculate? field) raw-answers
     (form-utils/numeric? field) (evenly-spaced-bins raw-answers 5 "int")
-    (form-utils/time-based? field) (evenly-spaced-bins raw-answers 5 "date")
+    (form-utils/time-based? field) (if (re-find #"^..:..$" (first raw-answers))
```

Comment:
why not push this into the function to DRY it up, `(evenly-spaced-bind raw-answers 5 (if ...))`